lib/src/Uncertainty/Model/DistributionFactoryImplementation.cxx

```diff
@@ -211,6 +241,10 @@ void DistributionFactoryImplementation::load(Advocate & adv)
 {
   PersistentObject::load(adv);
   adv.loadAttribute("bootstrapSize_", bootstrapSize_);
+  if (adv.hasAttribute("knownParameterValues_"))
+    adv.loadAttribute("knownParameterValues_", knownParameterValues_);
+  if (adv.hasAttribute("knownParameterIndices_"))
+    adv.loadAttribute("knownParameterIndices_", knownParameterIndices_);
```

Comment:
these sould appear in the save method too

Reply:
done